docker-compose.dev.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_PASSWORD=admin
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/postit
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+    environment:
```

Comment:
instead of this you could just define the env_file, gives a cleaner look. https://docs.docker.com/compose/compose-file/05-services/#env_file